BraveWallet/Preview Content/MockKeyringService.swift

```diff
@@ -332,4 +298,25 @@ class TestKeyringController: NSObject, BraveWalletKeyringController {
   func hasPendingUnlockRequest(_ completion: @escaping (Bool) -> Void) {
     completion(false)
   }
+  
+  func setKeyringDerivedAccountName(_ keyringId: String, address: String, name: String, completion: @escaping (Bool) -> Void) {
+    completion(false)
+  }
+  
+  func setKeyringImportedAccountName(_ keyringId: String, address: String, name: String, completion: @escaping (Bool) -> Void) {
+    completion(false)
+  }
```

Comment:
These previously had mock implementations, do we not need them?

Reply:
seems like unit tests are not using those methods. 
But i have add them back with the new `keyrings` i added